nextflow_schema.json

```diff
@@ -384,6 +384,10 @@
         },
         "malt_generatemegansummary": {
             "type": "boolean"
+        },
+        "kaiju_taxon_name": {
+            "type": "string",
+            "default": "species"
```

Comment:
Please give the range of taxa using the enumerated list option (accessable in `schema build` by pressing I think the gear/cog icon to the right of the given param entry)

```
   -r STRING     Taxonomic rank, must be one of: phylum, class, order, family, genus, species

```

Reply:
```
"kaiju_taxon_name": {
            "type": "string",
            "default": "species",
            "enum": [
                "phylum",
                "class",
                "order",
                "family",
                "genus",
                "species"
            ]
```

Shall I remove the default value then, right?

Reply:
No keep the default, I think that's fine